apps/els_lsp/src/els_unused_includes_diagnostics.erl

```diff
@@ -120,3 +121,31 @@ inclusion_range(Uri, Document) ->
     _ ->
       #{from => {1, 1}, to => {2, 1}}
   end.
+
+%% @doc Given a list of included uris, filter out the ones containing
+%% compiler attributes. If the Uri cannot be found, keep it in the list.
```

Comment:
to me it looks like if lookup_document fails the Uri is filtered out.

Reply:
Nice catch. Fixed (the code, not the comment).